code/API_definitions/Traffic Influence/Traffic_Influence.yaml

```diff
@@ -356,12 +354,10 @@ components:
         apiConsumerId:
             type: string
             description: Unique Identifier of the TI API Consumer.
-        applicationId:
-          type: string
-          example: "Virtual_Reality_Arena"
-          description:  Unique ID representing the Edge Application
-        instanceId:
-            $ref: '#/components/schemas/InstanceIdentifier'
+        AppId:
+            $ref: '#/components/schemas/AppId'

```

Comment:
Change in line 357 AppId for appId, avoiding starting with capital letter as in the rest of parameters.